### PR TITLE
XRDDEV-1770 Update tables

### PR DESCRIPTION
**Pull request by Riippi**
https://jira.niis.org/browse/XRDDEV-1770

### CODE REVIEW

File: src/centralserver/admin-ui/src/views/Members/Member/Details/MemberDetails.vue [hunk 1]
```diff
@@ -75,10 +75,34 @@
...
+          :search="searchServers"
+          :must-sort="true"
+          :items-per-page="-1"
+          class="elevation-0 data-table"
+          item-key="id"
+          :loader-height="2"
+          hide-default-footer
+        >
+          <template #[`item.namef`]="{ item }">
+            <div class="members-table-cell-name" @click="toDetails('netum')">
```
Comment (carohauta) [inline src/centralserver/admin-ui/src/views/Members/Member/Details/MemberDetails.vue:93]: You mentioned that some static data was left as is for now. Is this `netum` here on purpose?

**Review by carohauta — COMMENTED** [review]
Build has failed for some reason

File: src/centralserver/admin-ui/src/views/Members/Member/Details/MemberDetails.vue (hunk 1 above)
Comment (Riippi) [reply src/centralserver/admin-ui/src/views/Members/Member/Details/MemberDetails.vue]: No and the whole template is unused. I deleted it now.

**Comment by Riippi** [comment]
Seems that the build started working.

**Review by carohauta — APPROVED** [review]
🦄

Some conflicts to resolve before merging

**Pull request merged**